<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shop</class>
 <widget class="QWidget" name="shop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(122, 92, 255, 255), stop:0.899038 rgba(154, 13, 202, 255));</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 11, 244);
border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buy">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(255, 255, 255);
background-color: rgb(147, 11, 244);
border-radius: 12px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Beaver fat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 11, 244);
border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 11, 244);
border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buy_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(255, 255, 255);
background-color: rgb(147, 11, 244);
border-radius: 12px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Lure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buy_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(255, 255, 255);
background-color: rgb(147, 11, 244);
border-radius: 12px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Barabulka</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buy_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(255, 255, 255);
background-color: rgb(147, 11, 244);
border-radius: 12px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Nib</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 11, 244);
border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(128, 76, 244);</string>
   </property>
   <property name="text">
    <string>400</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(139, 50, 227);</string>
   </property>
   <property name="text">
    <string>400</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(137, 56, 231);</string>
   </property>
   <property name="text">
    <string>400</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(146, 34, 216);</string>
   </property>
   <property name="text">
    <string>400</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(50, 50, 50);
border-color: rgb(255, 255, 255);
background-color: rgb(144, 0, 255);
border-radius: 24px;
color: rgb(0, 0, 0)</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../build-untitled-Desktop_Qt_6_0_1_MinGW_64_bit-Debug/menu.png</normaloff>../../build-untitled-Desktop_Qt_6_0_1_MinGW_64_bit-Debug/menu.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
